docker-bake.hcl

```diff
@@ -22,6 +22,14 @@ target "api" {
   dockerfile = "build/new/api-server.Dockerfile"
   platforms = ["linux/arm64", "linux/amd64"]
   args = {
+    VERSION = "${VERSION}"
```

Comment:
I might actually need to define all these variables as I did for the ones above at line 4-12

Reply:
Can confirm, you must define all these variables before you can inject them through env variables

```
$ docker buildx bake --set api.platform=linux/arm64 --set api.args.GOCACHE="/Users/wito/Library/Caches/go-build" --set api.args.GOMODCACHE="/Users/wito/go/pkg/mod" api

ERROR: docker-bake.hcl:26,18-25: Unknown variable; There is no variable named "GIT_SHA"., and 7 other diagnostic(s)
```